coredns/tests/common.py

```diff
@@ -79,12 +79,13 @@
     NAMESPACE + '.forward_sockets_open',
 ]
 
-METRICS_V1_7 = COMMON_METRICS + [
+METRICS_V1_8 = COMMON_METRICS + [
     NAMESPACE + '.forward_max_concurrent_rejects',
     NAMESPACE + '.go.info',
     NAMESPACE + '.go.memstats.gc_cpu_fraction',
     NAMESPACE + '.go.memstats.heap_released_bytes',
+    NAMESPACE + '.cache_request_count',
     NAMESPACE + '.plugin_enabled',
```

Comment:
```suggestion
    NAMESPACE + '.plugin_enabled',
    NAMESPACE + '.forward_healthcheck_broken_count',
    NAMESPACE + '.hosts.reload_timestamp',
    NAMESPACE + '.reload.failed_count',
```
These metrics are now emitted with v1.8.5